backend/server.js

```diff
@@ -1,7 +1,14 @@
 import express from 'express';
+import mongoose from 'mongoose';
 import data from './data.js';
+import userRouter from './routers/userRouter.js';
 
 const app = express();
+mongoose.connect(process.env.MONGODB_URL || 'mongodb://localhost/amazona', {
+  useNewUrlParser: true,
+  useUnifiedTopology: true,
+  useCreateIndex: true,
+});
```

Comment:
useNewUrlParser, useUnifiedTopology, useFindAndModify, and useCreateIndex are no longer supported options....Removed

Reply:
You only need to remove "useCreateIndex: true", and the code would work perfectly fine.